src/utils/prefix.js

```diff
@@ -0,0 +1,13 @@
+const { getInput } = require("./action");
+
+/**
+ * Returns the prefix to use based on settings.
+ * @param {string} linter - the name of the linter.
+ * @returns {string} - the prefix to add to the command.
+ */
+function prefix(linter) {
```

Comment:
Please use verbs for function names whenever possible, e.g. `getPrefix`, or possibly `getCommandPrefix` for clarity and consistency with the option name.

Reply:
Ah sorry - I'm still living in Ruby-land. :)